Assets/LeapMotion/Scripts/HandPool.cs

```diff
@@ -51,23 +54,37 @@ public class ModelGroup {
       public List<IHandModel> modelList;
       public List<IHandModel> modelsCheckedOut;
       public bool IsEnabled;
+      public bool CanDuplicate;
+
+      public ModelGroup(string groupName, List<IHandModel> modelList, HandPool handPool, bool isEnabled, bool canDuplicate) {
+        this.GroupName = groupName;
+        this.IsEnabled = isEnabled;
+        this.CanDuplicate = canDuplicate;
+        this.modelList = modelList;
+        this.modelsCheckedOut = new List<IHandModel>();
+        this._handPool = handPool;
+      }
+
       public IHandModel TryGetModel(Chirality chirality, ModelType modelType) {
         for (int i = 0; i < modelList.Count; i++) {
-          if (modelList[i].Handedness == chirality && modelList[i].HandModelType == modelType) {
+          if (modelList[i].Handedness == chirality && modelList[i].HandModelType == modelType
```

Comment:
This conditional can be simplified to be
modelList[i].HandModelType == modelType && 
(modelList[i].Handedness == chirality || modelList[i].Handedness == Chirality.Either)